gh-actions/go/gotestfmt/action.yaml

```diff
@@ -0,0 +1,35 @@
+name: Install gotestfmt and our wrapper script
+description: Install gotestfmt and our wrapper script which improves the output and copies it to log files
+
+inputs:
+  tools-directory:
+    description: Directory pointing to go.mod file for checking tool versioning. If none is provided, the latest version will be downloaded.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install gotestfmt
+      working-directory: ${{ inputs.tools-directory }}
+      run: |
+        set -eu
```

Comment:
I think this match an older version of the script run, but if you look at https://github.com/ubuntu/authd-oidc-brokers/actions/runs/11820452458/job/32938431334#step:5:6, it seems this is this `set -eu`, no? (it’s the reason why I asked on the other run to have the comment in the shell script before this flag, so that it prints it). I have no idea why they don’t just print the job name itself as there is one.

That will avoid in a case we have a bunch of script to always end up with a list of `set -eu` as group title :)